.gitignore

```diff
@@ -6,3 +6,5 @@ vendor
 composer.lock
 build/
 swiftmailer/
+.idea
+.idea/
```

Comment:
This changes will lower the SensioLabs score, can you please remove it?
You should use a global .gitIgnore for non project-related files like those; see https://help.github.com/articles/ignoring-files/